cmd/soroban-cli/src/print.rs

```diff
@@ -36,6 +38,19 @@ impl Print {
         }
     }
 
+    // Some terminals like vscode's and macOS' default terminal will not render
+    // the subsequent space if the emoji codepoints size is 2; in this case,
+    // we need an additional space.
+    pub fn compute_emoji<T: Display + Sized>(&self, emoji: T) -> String {
+        if let Ok(term_program) = env::var("TERM_PROGRAM") {
```

Comment:
I'm not sure if this is a reliable way of telling the terminal. This will work for macos, but won't work for Linux (we have `$TERM`), and I'm not sure what the windows terminal uses. 
Wouldn't it be easier to just always print couple spaces in between emoji and main text? We always follow a pattern of `<emoji> <text message>` anyway